lib/web_ui/test/ui/fallback_fonts_golden_test.dart

```diff
@@ -293,6 +309,46 @@ void testMain() {
       await checkDownloadedFamilyForCharCode(0x1f3d5, 'Noto Color Emoji');
     });
 
+    // These code points exist in all of our CJK fonts.
+    final commonCJKCodePoints = <int>[
+      0x56de, // CJK Unified Ideographs
+      0x700b, // CJK Unified Ideographs
+      0x9c57, // CJK Unified Ideographs
+    ];
```

Comment:
I don't think testing in more than one codepoint adds much to the test? This is testing the language selection mostly, right?

Reply:
The original idea was to find code points that would naturally default to different fonts when no language is specified (e.g. one code point that defaults to JP and another that would default to KR). But I couldn't find any unfortunately.

I think I'll use just one code point and get rid of the `for` loop.